metadata_service/__init__.py

```diff
@@ -7,26 +7,29 @@
 import logging.config
 import os
 import sys
-from typing import Dict, Any  # noqa: F401
-from flask_cors import CORS
+from typing import Any, Dict  # noqa: F401
 
 from flasgger import Swagger
-from flask import Flask, Blueprint
+from flask import Blueprint, Flask
+from flask_cors import CORS
 from flask_restful import Api
 
+from metadata_service.api.badge import BadgeAPI
 from metadata_service.api.column import ColumnDescriptionAPI
-from metadata_service.api.dashboard import (DashboardDetailAPI, DashboardDescriptionAPI,
-                                            DashboardTagAPI, DashboardBadgeAPI)
+from metadata_service.api.dashboard import (DashboardBadgeAPI,
```

Comment:
Maybe we could mąke things consistent regarding wrapping modules in tuple (now there is mix of both w/ and w/o) ? I am in favor of tuple and multiline (one module per line) imports as it reduces chances for clashes during merges and makes things może readable in overall.

Reply:
@mgorsk1 I will add the isort to all repos.

Reply:
oh, I need to take a look at and see how to change isort for one module at a time though. I think it only handles import order.